airbyte-ci/connectors/metadata_service/lib/metadata_service/commands.py

```diff
@@ -26,28 +26,30 @@ def metadata_service():
 
 
 @metadata_service.command(help="Validate a given metadata YAML file.")
-@click.argument("file_path", type=click.Path(exists=True, path_type=pathlib.Path))
-def validate(file_path: pathlib.Path):
-    file_path = file_path if not file_path.is_dir() else file_path / METADATA_FILE_NAME
+@click.argument("metadata_file_path", type=click.Path(exists=True, path_type=pathlib.Path))
+@click.argument("doc_path", type=click.Path(exists=True, path_type=pathlib.Path))
```

Comment:
❗️ We should mark this required

Reply:
and we need to update the readme with an example https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connectors/metadata_service/lib/README.md?plain=1#L35

Reply:
I've marked the metadata-file-path, doc-path, and bucket-name arguments as required for the `validate` and `upload` commands, and I've updated the validate example in the readme